.docker/autoline/docker-compose.dev.yml

```diff
@@ -1,74 +1,54 @@
 version: "3.8"
```

Comment:
Why is this `docker-compose` called `.dev`? There's no reloading of the code on change. We even use the remote images

Reply:
dev because of dev server. If we want up docker localy we can just leave docker-compose.yml or docker-compose.local